forge-game/src/main/java/forge/game/trigger/TriggerDrawn.java

```diff
@@ -87,6 +87,13 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
             }
         }
 
+        if (hasParam("FirstCardInDrawStep")) {
```

Comment:
you could refactor it so both True and False work in favor of `NotFirstCardInDrawStep`